Dockerfile

```diff
@@ -1,34 +1,77 @@
-FROM ubuntu:focal as app
+FROM ubuntu:focal as base
 
+# System requirements.
 RUN apt update && \
-  apt-get install -y software-properties-common && \
-  apt-add-repository -y ppa:deadsnakes/ppa && apt-get update && \
-  apt install -y git-core language-pack-en python3.8-dev python3.8-venv libmysqlclient-dev libffi-dev libssl-dev build-essential gettext openjdk-8-jdk && \
```

Comment:
it looks like not all of these packages were copied over in the new format, is that intentional (i.e. `libffi-dev`)

Reply:
Yeah thats intentional. I couldn't find any use of these packages e.g `gettext`, `libffi-dev`, `git-core`, `openjdk` and others so I removed them.

Reply:
@aht007 Could you explain to us how you arrived at the conclusion that these are not used? Did you bring up the service and test through the newly built docker image?

Reply:
It is both testing the new image as well as learning from other images as well. These packages were also present in Insights' Ansible roles but I removed them from there as well since I couldn't find their usage. Also as per our knowledge many of these packages are present in Ansible roles and docker files for historical reasons but they aren't used now so we are better off without them.
I have tried bringing up the container and testing the image and it looked fine but if someone from your team knows about any of the removed package's usage then I will be happy to put it back there.